icons/keyboard-off.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Diottodev",
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "unkeys"
```

Comment:
This should include at least all the tags from the `keyboard` icon. And maybe we can think of some more.